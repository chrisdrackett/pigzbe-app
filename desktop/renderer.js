import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
