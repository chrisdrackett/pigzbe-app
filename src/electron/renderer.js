import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
