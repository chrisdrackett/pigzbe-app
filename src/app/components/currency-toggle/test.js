import React from 'react';
import renderer from 'react-test-renderer';
import CurrencyToggle from './';

const props = {
    currency: 'GBP',
    onCurrencyChange: () => {}
};

describe('CurrencyToggle', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<CurrencyToggle {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
