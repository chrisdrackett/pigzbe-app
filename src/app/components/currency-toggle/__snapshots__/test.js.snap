// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrencyToggle renders correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "rgb(127, 153, 188)",
      "borderRadius": 22.5,
      "height": 45,
      "opacity": 1,
      "position": "relative",
      "width": 80,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgb(0, 50, 120)",
        "borderRadius": 20.5,
        "height": 41,
        "left": 2,
        "position": "absolute",
        "shadowColor": "rgb(0, 0, 0)",
        "shadowOffset": Object {
          "height": 2,
          "width": 1,
        },
        "shadowOpacity": 0.3,
        "shadowRadius": 2,
        "top": 2,
        "width": 41,
      }
    }
  />
  <Image
    source={
      Object {
        "testUri": "../../../src/app/components/currency-toggle/images/wollo.png",
      }
    }
    style={
      Object {
        "height": 13,
        "left": 14,
        "position": "absolute",
        "top": 16,
        "width": 16,
      }
    }
  />
  <View
    style={
      Object {
        "position": "absolute",
        "right": 17,
        "top": 13,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "rgb(255, 255, 255)",
          "fontSize": 16,
          "fontWeight": "bold",
        }
      }
    >
      Â£
    </Text>
  </View>
</View>
`;

exports[`TextInput renders correctly 1`] = `
Array [
  <Text
    style={
      Object {
        "color": "rgb(0, 50, 120)",
        "fontFamily": "Poppins",
        "fontSize": 16,
        "fontWeight": "bold",
        "marginBottom": 10,
        "marginLeft": 2,
        "marginTop": 10,
      }
    }
  >
    Label
  </Text>,
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "height": 55,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="sentences"
      autoCorrect={true}
      blurOnSubmit={true}
      editable={true}
      keyboardType="default"
      multiline={false}
      numberOfLines={1}
      onChangeText={[Function]}
      placeholderTextColor="rgb(127, 153, 188)"
      returnKeyType="done"
      style={
        Array [
          Object {
            "alignSelf": "stretch",
            "borderColor": "rgb(127, 153, 188)",
            "borderRadius": 22.5,
            "borderWidth": 1,
            "color": "rgb(0, 50, 120)",
            "fontFamily": "Poppins",
            "fontSize": 14,
            "fontWeight": "bold",
            "lineHeight": 21,
            "marginBottom": 10,
            "paddingLeft": 20,
            "paddingRight": 20,
            "paddingTop": 12,
            "textAlignVertical": "top",
          },
          Object {
            "borderRadius": 22.5,
            "height": 45,
            "paddingTop": 0,
            "width": "100%",
          },
        ]
      }
      underlineColorAndroid="transparent"
      value=""
    />
  </View>,
]
`;
